common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -161,8 +162,16 @@
                     },
                     error: DiscussionUtil.formErrorHandler(this.$('.post-errors')),
                     success: function(response) {
-                        var thread;
+                        var thread, discussionBoardModel;
```

Comment:
My point is that you are assigning `self.discussionBoardView.breadcrumbs.model` to the `discussionBoardModel ` so giving such name is not suitable because it's not board model. It's board's breadcrumbs model. You may want to change its name that is more appropriate.